code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -67,3 +67,13 @@
 /obj/item/gun/energy/disabler/cyborg/newshot()
 	..()
 	robocharge()
+
+/obj/item/gun/energy/disabler/silencer
+	name = "u-ION Silencer"
```

Comment:
is this name supposed to be a reference to something? its not an ion gun, seems confusing

Reply:
Joke on union silencer,  NT having a way to "naturally" kill off their competition seems like something they would do, however ION in it is problematic. Open for other name suggestions

Reply:
@hal9000PR can you say if this comment is resolved or not? Code looks good as is

Reply:
@farie82 yeah i don't have any better ideas.